cohttp-eio/examples/client1.ml

```diff
@@ -0,0 +1,16 @@
+open Eio
+open Cohttp_eio
+
+let conn env sw () =
+  let hostname, port = ("www.example.org", 80) in
+  let he = Unix.gethostbyname hostname in
+  let addr = `Tcp (Eio_unix.Ipaddr.of_unix he.h_addr_list.(0), port) in
+  let flow = (Net.connect ~sw (Stdenv.net env) addr :> Eio.Flow.two_way) in
```

Comment:
Are these casts needed? Probably it indicates that the API is too strict if users need to do this.

Reply:
It doesn't need the cast now.